enterprise/frontend/src/metabase-enterprise/smtp-override/components/CloudSMTPConnectionCard.tsx

```diff
@@ -53,7 +59,9 @@ export const CloudSMTPConnectionCard = ({
                 )}
 
                 <div>
+                  {/* eslint-disable-next-line no-literal-metabase-strings -- Not user configurable */}
                   <Text className={S.label}>{t`Managed by Metabase`}</Text>
+                  {/* eslint-disable-next-line no-literal-metabase-strings -- Not user configurable */}
```

Comment:
not sure what "not user configrable" means - usually the reason we give is that we can show the name metabase in admin settings